api/src/lib.rs

```diff
@@ -610,11 +610,15 @@ impl Client {
         Ok(())
     }
 
-    pub fn get_statistics(&self, dataset_name: &DatasetFullName) -> Result<Statistics> {
+    pub fn get_statistics(
+        &self,
+        dataset_name: &DatasetFullName,
+        params: &StatisticsRequestParams,
+    ) -> Result<Statistics> {
         Ok(self
             .post::<_, _, GetStatisticsResponse>(
                 self.endpoints.statistics(dataset_name)?,
-                json!({}),
+                json!(params),
```

Comment:
While this does what you want, it's better to use `serde_json::to_value` which returns a `Result` (indicating the result is fallible). You can then unwrap/expect/`?` this as you want.